src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/235892484-cc503121-cbcf-4645-8971-62625ad50af7.png)
It is necessary to move Random to a class variable, as in this example from the checklist